src/main/kotlin/com/atlassian/performance/tools/virtualusers/api/EntryPoint.kt

```diff
@@ -28,12 +33,22 @@ class Application {
     }
 
     private fun run(args: Array<String>) {
-        val options = VirtualUserOptions.Parser().parse(args)
-        @Suppress("DEPRECATION")
-        if (options.help) {
-            options.printHelp()
-        } else {
-            LoadTest(options, RestUserGenerator()).run()
-        }
+        val options: VirtualUserOptions = VirtualUserOptions.Parser().parse(args)
+
+        setLoggingConfiguration(options)
+
+        if (options.help) options.printHelp()
+        else LoadTest(options, RestUserGenerator()).run()
     }
-}
\ No newline at end of file
+
+    private fun setLoggingConfiguration(options: VirtualUserOptions) {
+        val logging = options.behavior.logging
+
+        val logConfig = logging.getConstructor().newInstance() as AbstractConfiguration
+        val logConfigurationFactory = LogConfigurationFactory(logConfig)
+        setConfigurationFactory(logConfigurationFactory)
+        val ctx = LogManager
+            .getContext(false) as org.apache.logging.log4j.core.LoggerContext
```

Comment:
Can we import `org.apache.logging.log4j.core.LoggerContext`?

Reply:
Sure